web/src/engine/FetchProviderNodeWebkit.ts

```diff
@@ -158,10 +158,10 @@ export async function FetchCSS<T extends HTMLElement>(request: FetchRequest, que
     return [...dom.querySelectorAll(query)] as T[];
 }
 
-export async function FetchGraphQL<TResult>(request: FetchRequest, operationName: string, query: string, variables: string): Promise<TResult> {
+export async function FetchGraphQL<TResult>(request: FetchRequest, operationName: string, query: string, requestVariables: string = undefined): Promise<TResult> {
 
     const graphQLRequest = new FetchRequest(request.url, {
-        body: JSON.stringify({ operationName, query, variables }),
+        body: JSON.stringify({ operationName, query, variables: requestVariables ? requestVariables: {} }),
```

Comment:
Type mismatch, `variables` is defined as string, but an empty object `{}` is used as default.

Reply:
IKR .If i use a string the request doesnt work. Teh request
Do you suggest i modify FetchGraphQL argument to pass an objet of type "unknown" so its settled ?

Reply:
Changes reverted, and so its gonna need #406 merged to work :)